samples/excel/89-preview-apis/worksheet-freeze-panes.yaml

```diff
@@ -0,0 +1,211 @@
+id: excel-89-preview-apis-worksheet-freeze-panes-
+name: Manage frozen panes in a worksheet
+description: Freeze and unfreeze columns and rows and manage frozen panes in a worksheet.
+author: OfficeDev
+host: EXCEL
+api_set:
+    ExcelApi: 1.7
+script:
+    content: |
+        $("#setup").click(() => tryCatch(setup));
+
+        $("#freeze-columns").click(() => tryCatch(freezeColumns));
+        $("#unfreeze-all-columns").click(() => tryCatch(unfreezeAllColumns));
+        $("#freeze-rows").click(() => tryCatch(freezeRows));
+        $("#unfreeze-all-rows").click(() => tryCatch(unfreezeAllRows));
+        $("#freeze-at").click(() => tryCatch(freezeAt));
+        $("#get-location").click(() => tryCatch(getLocation));
+        $("#unfreeze-all-panes").click(() => tryCatch(unfreezeAllPanes));
+
+        async function freezeColumns() {
+            await Excel.run(async (context) => {
+                const activeSheet = context.workbook.worksheets.getActiveWorksheet();
```

Comment:
Active sheet or get sheet by name?

Reply:
Good call -- I probably should get sheet by name, since the "setup" function explicitly creates a "Sample" sheet. I'll make this change in my other snippets for this as well.